android/app/src/main/java/app/organicmaps/MwmActivity.java

```diff
@@ -227,6 +237,14 @@ public interface LeftAnimationTrackListener
     void onTrackLeftAnimation(float offset);
   }
 
+  public enum PowerSaveDisclaimerState
```

Comment:
since disabling power save mode is very important feature so i think its better to ask users once every session to make them aware. Ask only once in life time was not a good idea and most probably user would have missed it because in first run alot of things are there to read.